tests/test_instance/server/settings.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python
```

Comment:
Probably don't need all these settings, might be good to stick with the default config

Reply:
you're right. Removed.